source/winVersion.py

```diff
@@ -28,36 +29,40 @@ def isUwpOcrAvailable():
 
 
 WIN10_VERSIONS_TO_BUILDS = {
-	1507: 10240,
-	1511: 10586,
-	1607: 14393,
-	1703: 15063,
-	1709: 16299,
-	1803: 17134,
-	1809: 17763,
-	1903: 18362,
-	1909: 18363,
-	2004: 19041,
-	2009: 19042,
+	"1507": 10240,
+	"1511": 10586,
+	"1607": 14393,
+	"1703": 15063,
+	"1709": 16299,
+	"1803": 17134,
+	"1809": 17763,
+	"1903": 18362,
+	"1909": 18363,
+	"2004": 19041,
+	"20H2": 19042,
 }
 
 
-def isWin10(version: int = 1507, atLeast: bool = True):
+def isWin10(version: Union[int, str] = 1507, atLeast: bool = True):

```

Comment:
I think this could become a mistake from and API sense. This will result in a very hard to untangle confusion between version and build.

It's also not clear on first glance (without reading the docs / implementation) that this function tells you if the OS is a particular version or greater or if it tells you that the versions string is a valid one for Windows 10.

To fix this it might be helpful to keep but deprecate the `version` keyword, and introduce `matchesVersionOrLater: str` and a `matchesBuildOrLater: int` keywords. Alternatively have two separate public functions (which could share a private implementation if it is appropriate). 

Another alternative, would be a new `getWinVer()` function that returned a class type that handled comparison operators and could also be constructed by a build string / int. Allowing you to write code like"

```
if winVer('20H2') < getWinVer() < winVer('21H2'):
    print("in expected range or releases")
```